### PR TITLE
Fixed responsiveness on product detail view and checkout summary

### PR DESCRIPTION
**Pull request by jaepass**
- Fixed small mobile view bugs on product detail view and checkout summary spotted by Andrew

### CODE REVIEW

File: pages/checkout/index.js [hunk 1]
```diff
@@ -449,7 +449,7 @@ class CheckoutPage extends Component {
                     )
                   })}
                 </div>
-                <form className="d-flex py-3 borderbottom border-color-gray400">
+                <form className="py-3 borderbottom border-color-gray400 discount">
```
Comment (john-raymon) [inline pages/checkout/index.js:452]: Think creating an additional discount class just to apply `display: flex` is redundant here when `d-flex` is available already providing that.
```suggestion
                <form className="d-flex py-3 borderbottom border-color-gray400">
```

File: style/scss/pages/_checkout.scss [hunk 2]
```diff
@@ -10,8 +10,29 @@
...
     top: 15%;
 
-    @include md-max {
+    @include md {
       width: 100%;
     }
+
+    .discount {
+      display: flex;
+    }
```
Comment (john-raymon) [inline style/scss/pages/_checkout.scss:19]:
```suggestion
```

File: pages/checkout/index.js (hunk 1 above)
Comment (jaepass) [reply pages/checkout/index.js:452]: yes originally had d-flex and d-sm-block but messed by the flex alignment on the input and button. i targeted in css to fix the display block bug on mobile view

File: style/scss/pages/_checkout.scss (hunk 2 above)
Comment (jaepass) [reply style/scss/pages/_checkout.scss:19]: same here, got rid of `d-flex` in this element and target in css with `.discount`

**Comment by jaepass** [comment]
applied boostrap's `col` classes as most elements in this repo are using this convention. applying flex and block was somehow getting rid of the button's height on small screen size.

**Pull request merged**